streamApp/Makefile

```diff
@@ -78,6 +78,13 @@ endif
 
 PROD_LIBS += $(EPICS_BASE_IOC_LIBS)
 
+# Some linux systems moved RPC related symbols to libtirpc
+# Define TIRPC in configure/CONFIG_SITE in this case
+ifeq ($(TIRPC),YES)
+  USR_INCLUDES_Linux += -I/usr/include/tirpc
+  PROD_SYS_LIBS_DEFAULT += tirpc
+endif
+
 # switch off annoying rset warnings in 3.16+
```

Comment:
I am still not convinced that "repackage the existing system libtirpc.a into libasyn.a" is not the better approach. When building libasyn.a (or even libasyn.so), it should be possible to statically link it with libtirpc.a, so that all rpc functions asyn needs are in the asyn lib. On the other hand, this change looks innocent enough.

Reply:
In static build of a library, there is actually no linking. Only at creating the executables or dynamic libraries, the symbols get resolved. The dependencies of asyn have to be propagated to the final build. Ideally this can be provided by asyn module as a snippet Makefile file and dependent modules include this file. This is used by the areaDetector modules.
https://github.com/areaDetector/ADSimDetector/blob/master/iocs/simDetectorIOC/simDetectorApp/src/Makefile#L26

Reply:
I have asked for something similar for asyn. Maybe in one of the future versions...